tools/lib/provision.py

```diff
@@ -325,6 +347,12 @@ def install_yum_deps(deps_to_install, retry=False):
     run_as_root(["cp", "-a", "puppet/zulip/files/postgresql/centos_pg_hba.conf", pg_hba_conf])
     # Later steps will ensure postgres is started
 
+    # Link in tsearch data files
+    overwrite_symlink("/usr/share/myspell/en_US.dic", "/usr/pgsql-%s/share/tsearch_data/en_us.dict"
```

Comment:
Is this now required for all distributions? Otherwise, since the scope is only for tsearch, I don't see why it should be here instead of in build-tsearch-extras.